src/data/attribute.rs

```diff
@@ -129,7 +130,42 @@ pub fn store_metadata(metastruct: &Struct) {
     let metadata = process_metadata(metastruct, String::new());
     for (key, value) in metadata {
         let attr = format!("{KUADRANT_NAMESPACE}\\.{key}");
+        // stored into host_property: wasm.kuadrant.{key}
```

Comment:
`wasm.kuadrant.{key}` or `kuadrant.{key}` ?

Reply:
`wasm.` prefix is automatically added by envoy. So you store `foobar`, but to retrieve it back you read `wasm.foobar`...